FlowCal/io.py

```diff
@@ -972,6 +976,17 @@ def analysis(self):
         return self._analysis
 
     @property
+    def data_type(self):
+        """
+        Type of data in the FCS file's DATA segment.
+
+        `data_type` is 'I' if the data type is integer, 'F' for floating
+        point, and 'D' for double.
+
+        """
+        return self._data_type
```

Comment:
There is no place in FlowCal in which we access `.text`, and I wanted to keep it that way.